index.ts

```diff
@@ -44,6 +45,12 @@ async function main() {
                 // if issue exists in devpool
                 const devpoolIssue = getIssueByLabel(devpoolIssues, `id: ${projectIssue.node_id}`);
                 if (devpoolIssue) {
+                    const issueDetails = await octokit.rest.issues.get({
+                        owner: ownerName,
+                        repo: repoName,
+                        issue_number: devpoolIssue.number,
+                    });
+                    const additionalLabelsToAdd = issueDetails.data?.assignee ?['Unavailable']:[]
                     // if title or state or pricing was updated then update a devpool issue
                     if (devpoolIssue.title !== projectIssue.title || devpoolIssue.state !== projectIssue.state || getIssuePriceLabel(devpoolIssue) !== getIssuePriceLabel(projectIssue)) {
```

Comment:
if assignee has been updated then the `Unavailable` label will appear/disappear only if title/state/price labels have changed, i.e. right now on assignee change the issue is not updated, pls fix

Reply:
fixed